tests/migration_test.go

```diff
@@ -437,6 +454,8 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 
 			It("[test_id:3237]should complete a migration", func() {
 				vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+				vmi = makeMigratableVMI(vmi)
```

Comment:
I don't understand: why do we create a new random vm and then do a deep copy and set one field? Why don't we just set the field?

Same goes for the other lines where this is done?

Reply:
Ah thanks @dhiller I forgot about this, it is not required. I copied snippet from different path and edited it. I removed it.